date1.java

```diff
@@ -0,0 +1,188 @@
+                        case "WED": {
+                            order[i] = 3;
+                            break;
+                        }
+                        case "THU": {
+                            order[i] = 4;
+                            break;
+                        }
+                        case "FRI": {
+                            order[i] = 5;
+                            break;
+                        }
+                        case "SAT": {
+                            order[i] = 6;
+                            break;
+                        }
+                        case "SUN": {
+                            order[i] = 7;
+                            break;
+                        }
+                    }
+                }
+
+                if (LstartDate.isAfter(systemTime)) {
+                    nextDate = LstartDate;
+                } else {
+                    LocalDateTime temp = LstartDate;
+                    if (!(temp.getDayOfWeek().getValue() <= order[dayList.length - 1]))
+                        temp = temp.plusWeeks(1);
+
+                    nextDate = systemTime.withHour(LstartDate.getHour()).withMinute(LstartDate.getMinute())
+                            .withSecond(LstartDate.getSecond());
+                    long noOfWeeksElapsed = ChronoUnit.WEEKS.between(temp.with(DayOfWeek.MONDAY),
+                            systemTime.with(DayOfWeek.SUNDAY));
+                    long dueWeeks = noOfWeeksElapsed % repeatEvery;
+
+                    if (dueWeeks == 0) {
+                        LocalDateTime temp1 = systemTime.withHour(LstartDate.getHour())
+                                .withMinute(LstartDate.getMinute()).withSecond(LstartDate.getSecond());
+                        if ((temp1.getDayOfWeek().getValue() <= order[dayList.length - 1]))
+                            nextDate = temp1;
+                        else
+                            nextDate = nextDate.with(DayOfWeek.MONDAY).plusWeeks(repeatEvery);
+                    } else {
+                        nextDate = nextDate.plusWeeks(repeatEvery - dueWeeks);
+                        nextDate = nextDate.with(DayOfWeek.MONDAY);
+                        nextDate = nextDate.plusDays(order[0] - 1);
+                    }
+                }
+                
+                
+                while(true)
+                {
+                    if(Arrays.stream(dayList).anyMatch(nextDate.getDayOfWeek().name().substring(0, 3)::equals) && nextDate.isAfter(systemTime))
+                    {
+                        if(LendDate.isAfter(nextDate))
+                            return Date.from(nextDate.atZone(currentZone).toInstant());
+                        else
+                            return null;
+                    }
+
+                    nextDate=nextDate.plusDays(1);
+                }
+            }
+
+
+            case Monthly:
+            {
+                int j=(runOnDay=="last")?0:Integer.parseInt(runOnDay);
```

Comment:
code indentation is missing. Make sure we clean the code and do proper indentation every time before we raise a pull request.
also I see variable name j.